README.md

```diff
@@ -18,15 +17,32 @@ operationally sound and effective workloads without understanding the underlying
 infrastructure.
 
 It makes extensive use of plugins. Thus, the type of code which it can accept is
-broad and covers languages such as Python 3 or golang, frameworks such as
-django, and actual applications such as Sphinx, Drupal or even Quake 3.
+broad and covers languages such as Python 3 or Go, 
+and actual applications such as Sphinx, Drupal or even Quake 3.
 
-These "job types", in Racetrack lingo, can be deployed to different
+These "jobs", in Racetrack lingo, can be deployed to different
 "infrastructure targets". We currently support Kubernetes and Docker. This is
 also pluggable and can be extended.
 
 Code deployed via Racetrack gets free Swagger pages and Prometheus metrics endpoints.
 
+What distinguishes Racetrack is:
+
+- **Server-side building**. Code is transformed into a microservice without your computer being involved.
+- You only supply your **function's logic**. No need to write repetitive API code, setting up 
+  webservers, creating dockerfiles, kubernetes YAMLs, and so on. Racetrack takes care of that for you.
+- **Language agnostic**. Deploy code written in Python 3, Go, Rust,
+  or anything else encapsulated in a Dockerfile.
+- **Infrastructure independent**. Deploy to either a Kubernetes cluster
+  or a Docker environment in a single transparent step.
+- **Customizable through plugins**,
+- **Immutable jobs**
+- **Reproducible jobs**. Racetrack ensures that anyone else can deploy the same job effortlessly.
+- **Out-of-the-box tools**: web endpoints, API documentation,
+  metrics, monitoring, tracing, and more.
+- Released under a **permissive open-source license**,
+- Suitable for both **on-premises and cloud** environments.
```

Comment:
I'd be inclined to just choose 3 points even if they are all valid.

When the list is long it becomes its own job to even read the blurb. Most people will lose focus. It's not your fault, it's Twitter and Facebook and Youtube and TikTok.

While it's hard to choose, consider this: you're trying to make sure that there's something on that list for everyone, but that makes the list long and then you get no one.

My feeling is, numbers 2 3 and 4.

Reply:
I agree, the list is quite long and 2,3,4 are the most catchy three. I'm going to move the rest to the User Guide doc (for more patient readers) cause I feel like this still explains important questions, but these questions will most likely arise at a later point.